packages/mobile/src/dappkit/DappKitSignTxScreen.tsx

```diff
@@ -121,24 +117,18 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
     marginHorizontal: '15%',
+    marginVertical: '50%',
```

Comment:
ditto about using `flexGrow: 1` to prevent content clipping on smaller screen sizes.